src/styles/annotator/components/Toolbar.scss

```diff
@@ -2,6 +2,8 @@
 
 @use '../variables' as var;
 
+@use '../../shared/components/buttons/buttons' as shared-buttons;
```

Comment:
This particular SASS module still needs to use the "old" button mixins for a little while until they can be updated, thus this namespacing.